release/models/policy/openconfig-routing-policy.yang

```diff
@@ -162,10 +177,16 @@ module openconfig-routing-policy {
   typedef policy-result-type {
     type enumeration {
       enum ACCEPT_ROUTE {
-        description "Policy accepts the route";
+        description "Policy accepts the route and evaluation of the
+        current policy definition stops.";
       }
       enum REJECT_ROUTE {
-        description "Policy rejects the route";
+        description "Policy rejects the route and evaluation of the
+        current policy definition stops.";
+      }
+      enum NEXT_STATEMENT {
+        description "Any modifications of the route are preserved and the evaluation of the policy will continue to the
+        next statement.";
       }
     }
```

Comment:
Add a default statement for this typedef?

Reply:
done